src/constants.ts

```diff
@@ -8,14 +8,24 @@ export enum COMMANDS {
   PARSER = 'PARSER',
   CUSTOM = 'CUSTOM',
 }
-export type FluentBitSchemaType = {
+
+export type FluentBitSection = {
   id: string;
   name?: string;
   command: COMMANDS;
-  optional?: Record<string, unknown>;
-} & { [key: string]: unknown };
+  optional?: { [key: string]: unknown };
```

Comment:
Can you just clarify what this is doing here?
An array of keys (made up of strings) mapping to an `unknown` value? Is `unknown` a specific type like `any`?
Ah, yes a special case for `any`: https://mariusschulz.com/blog/the-unknown-type-in-typescript